packages/web-react/src/components/TextField/demo/TextFieldDefault.tsx

```diff
@@ -5,15 +5,23 @@ const TextFieldDefault = () => (
   <>
     <TextField id="textfield-default" label="Label" name="textfieldDefault" placeholder="Placeholder" />
 
-    <TextField id="textfield-filled" label="Label" name="textfieldFilled" placeholder="Placeholder" value="Filled" />
+    <TextField
+      id="textfield-filled"
+      label="Label"
+      name="textfieldFilled"
+      onChange={(e) => console.log(`Value: ${(e.target as HTMLInputElement).value}`)}
+      placeholder="Placeholder"
+      value="Filled"
```

Comment:
Replace with `value`, so `onChange` doesn't have to be defined. Or use states to be able to change the input value.

Reply:
@pavelklibani had a good point that it's not possible to change the value right now, so that's the reason why I would like to fix it and use `defaultValue` (so `onChange` doesn't have to be defined).